package.json

```diff
@@ -37,6 +37,7 @@
     "express": "^4.16.3",
     "express-session": "^1.15.6",
     "fs-extra": "^7.0.0",
+    "gemfile": "github:marcandre/gemfile",
```

Comment:
looks like the maintainer is responsive and we will be able to get rid of that quickly

https://github.com/treycordova/gemfile/pull/2